flagsmith/provider.go

```diff
@@ -9,15 +9,16 @@ import (
 
 	"github.com/hashicorp/terraform-plugin-framework/diag"
 	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
```

Comment:
Do we still need to include tfsdk as a dependency now?

Reply:
Yep, it's still being used, e.g: https://github.com/Flagsmith/terraform-provider-flagsmith/pull/32/files#diff-394e3c67cba2e4d057cb7156c56e7fd9e39a433cf41935a78fe014b600bfaa59R92 `tfsdk.Schema`